app/styles/_uswds-theme.scss

```diff
@@ -0,0 +1,19 @@
+/*
+----------------------------------------
+USWDS with settings overrides
+----------------------------------------
+Uncomment the following lines and add a list of changed settings
+in the form $setting: value,
+----------------------------------------
+*/
+
+@use 'uswds-core' with (
+  /* Disable scary but mostly irrelevant warnings: */
+    $theme-show-compile-warnings: false,
+  $theme-show-notifications: false,
+
+  // Specify USWDS font and image paths.
+  // Same paths are successfully used for both Next.js and Storybook.
+  $theme-font-path: '/uswds/fonts',
+  $theme-image-path: '/uswds/img'
```

Comment:
These static font and image assets are showing 404's for me currently in the app. I've only ran in Docker so far.

<img width="1066" alt="CleanShot 2022-07-11 at 15 25 02@2x" src="https://user-images.githubusercontent.com/371943/178369114-2ffed015-3d7f-42a5-a672-e69e5d2a9a4a.png">

Relatedly, I see the following warning when I run Storybook in Docker:

<img width="720" alt="CleanShot 2022-07-11 at 15 34 10@2x" src="https://user-images.githubusercontent.com/371943/178370013-69126592-1af1-40a0-96e9-f705ece3d33c.png">

Reply:
@sawyerh Ok I think I've solved this by using a named volume for `node_modules`. This means that _almost_ all of the files in `app` are bind mounted into the docker container. I've excluded `node_modules` so that the container uses the `node_modules` inside the container instead of whatever was set up the last time `yarn install` was run on the host machine.